### DISCUSSION TITLE
Error: Int.default is not a function

### DISCUSSION THREAD

**Discussion by farrukh-tsf** [storybookjs/storybook#28976] [Help]
### Summary

while installing story book to a current project I am getting following error
**Int.default is not a function**

![image](https://github.com/user-attachments/assets/c296907f-0766-4c7a-ba73-941c3d8acbf5)

### Additional information

Project Details:

Vue 3 + VITE + JS

Node: 20.17.0

![image](https://github.com/user-attachments/assets/9c07257d-395c-467d-858d-6e6bda41a021)

**Comment by mhornbacher** [comment]
Did you happen to find a workaround?
